src/redux/reducers/index.js

```diff
@@ -2,14 +2,14 @@ import { combineReducers } from 'redux';
 import LandingPageReducer from './landingPageReducer';
 import authReducer from './authReducer';
 import passwordResetReducer from './passwordResetReducer';
-import socialReducer from './socialReducer';
+// import socialReducer from './socialReducer';
 import singleArticleReducer from './articleReducer';
 
 
 export default combineReducers({
   article: LandingPageReducer,
   auth: authReducer,
   requestReset: passwordResetReducer,
-  social: socialReducer,
+  // social: socialReducer,
```

Comment:
I don't have leave a comment for every instance where code became a comment

Reply:
sorry about that. I'll fix it now

Reply:
Done that